src/stylesheets/application.scss

```diff
@@ -0,0 +1,9 @@
+@import 'bootstrap_manifest';
```

Comment:
@rafayet-monon I think it better to use just `bootstrap` instead of `bootstrap_manifest`.
What do you think?

Reply:
Changed! 🚀